wiki/Gameplay/Hit_object/Slider/es.md

```diff
@@ -1,5 +1,4 @@
 ---
-no_native_review: true
```

Comment:
this was set on your own translation, I think it should take another user's review to get it removed